policy_engine: rename inbound/outbound to new phase names (#955)
ngrok/ngrok-docs: traffic-policy/actions/add-headers/behavior.mdx

## Behavior

> When configured in the `on_http_request` phase, this action will add headers on an incoming http request before reaching the upstream server with
> the configured headers. When configured in the `on_http_response` phase, the configured headers are added to the response from the upstream server.

Reviewer: ```suggestion
When configured in the `on_http_request` phase, this action will add the specified headers to incoming http requests before reaching the upstream server. When configured in the `on_http_response` phase, the specified headers are added to the http response from the upstream server.
```
